src/components/side_panel/chart/building_blocks/axis_design/axis_design_editor.ts

```diff
@@ -213,4 +226,43 @@ export class AxisDesignEditor extends Component<Props, SpreadsheetChildEnv> {
     const type = this.props.definition.type;
     return type === "line" || type === "scatter";
   }
+
+  private parseAxisBoundaryValue(ev: InputEvent): number | undefined | null {
+    const input = ev.target as HTMLInputElement;
+    const value = input.value.trim();
+    if (value === "") {
+      return undefined;
+    }
+    if (this.isTimeAxis) {
+      const timestamp = this.getTimestampFromInput(input);
```

Comment:
You should really convert that to a spreadsheet date number (`1` is 1/1/0900) - not a js timestamp. If I add a limit then remove the date format, I get an huge value that doesn't make sense (eg. `1756425600000`). If you use spreadsheet date numbers, it would make sense even after removing the format.